integration-cli/docker_cli_run_test.go

```diff
@@ -144,18 +144,18 @@ func TestRunLeakyFileDescriptors(t *testing.T) {
 	logDone("run - check file descriptor leakage")
 }
 
-// it should be possible to ping Google DNS resolver
+// it should be possible to lookup Google DNS
 // this will fail when Internet access is unavailable
-func TestRunPingGoogle(t *testing.T) {
+func TestRunLookupGoogleDns(t *testing.T) {
 	defer deleteAllContainers()
 
-	runCmd := exec.Command(dockerBinary, "run", "busybox", "ping", "-c", "1", "8.8.8.8")
-	out, _, _, err := runCommandWithStdoutStderr(runCmd)
+	runCmd := exec.Command(dockerBinary, "run", "busybox", "sh", "-c", `[[ -n "$(nslookup 8.8.8.8 | grep google.com)" ]]`)
+	out, _, ec, err := runCommandWithStdoutStderr(runCmd)
 	if err != nil {
-		t.Fatalf("failed to run container: %v, output: %q", err, out)
+		t.Fatalf("failed to run container: %v, exitCode=%d, output: %q", err, ec, out)
```

Comment:
@ahmetalpbalkan you won't like my comment but the exitCode is redundant since `err` will contain the exitcode in its message.